widget/ui/src/icons/Exchange.tsx

```diff
@@ -0,0 +1,22 @@
+import type { SvgIconPropsWithChildren } from '../components/SvgIcon';
+
+import React, { createElement } from 'react';
+
+import { SvgIcon } from '../components/SvgIcon';
+
+function SvgExchange(props: SvgIconPropsWithChildren) {
+  return createElement(
+    SvgIcon,
+    props,
+    <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+      <g id="Icons">
+        <path
+          id="Vector"
```

Comment:
Do we need this id?

Reply:
The file related to the icons is created automatically and this ID is probably in the SVG file